lightning/src/chain/mod.rs

```diff
@@ -299,6 +299,9 @@ pub trait Watch<ChannelSigner: WriteableEcdsaChannelSigner> {
 	/// Update the outpoint funding the channel.
 	fn update_channel_funding_txo(&self, old_funding_txo: OutPoint, new_funding_txo: OutPoint, channel_value_satoshis: u64) -> ChannelMonitorUpdateStatus;
 
+	/// Get the latest commitment transaction to broadcast
+	fn get_latest_holder_commitment_txn(&self, funding_txo: OutPoint) -> Vec<bitcoin::Transaction>;
```

Comment:
Since the point of getting the commitment transactions is to close the channel, and that IIUC the HTLC transactions are also required to properly close it I think it make sense to have them as well.

Reply:
Yep, I agree it makes sense. The naming is a bit confusing, but I think that was also the case in the main fork.